akka-stream/src/main/resources/reference.conf

```diff
@@ -47,6 +47,9 @@ akka {
     # such as FileSource, FileSink and others.
     file-io-dispatcher = "akka.stream.default-file-io-dispatcher"
 
+    # Blocking IO like OutputStreamSource and InputStreamSink uses by default the same settings
+    blocking-io-dispatcher = "akka.stream.default-file-io-dispatcher"
```

Comment:
+1, let's just make sure it uses the configured (by `withAttributes`) dispatcher, and no default then?